tests/generators/test_rest.py

```diff
@@ -122,3 +122,37 @@ def test_rest_skip_code(requests_mock):
     )
     output = generator._call_model("Who is Enabran Tain's son?")
     assert output == [None]
+
+
+@pytest.mark.usefixtures("set_rest_config")
+@pytest.mark.parametrize("verify_ssl", (True, False, None))
+def test_rest_ssl_suppression(mocker, requests_mock, verify_ssl):
+    if verify_ssl is not None:
```

Comment:
when can it be `None`?

Reply:
`verify_ssl` is from the test cases in line 128, `None` handling here is to allow a single test to check the default value is held when the user does not provide a value for `verify_ssl` via configuration and to validate each supported user value.